elevation_mapping:
  ros__parameters:
    map_frame: "map"
    camera_frame: "camera_depth_frame"
    resolution: 0.1
    width: 60
    depth_topic: "/depth_camera/points"
    min_z: 0.1
    max_z: 1.0
    inflation_radius: 0.7
    inflate_obstacles: true
